src/include/wiredtiger.in

```diff
@@ -1897,6 +1907,25 @@ struct __wt_session {
 	int __F(timestamp_transaction)(WT_SESSION *session, const char *config);
 	/*! @} */
 
+	/*!
+	 * Set the commit timestamp on a transaction numerically.  Prefer this over @ref
+	 * timestamp_transaction when the string parsing done in that method becomes a bottleneck.
+	 *
+	 * The WT_SESSION.timestamp_transaction_numeric method can only be used at snapshot
+	 * isolation.
+	 *
+	 * @requires_transaction
+	 *
+	 * @param session the session handle
+	 * @param which the timestamp you wish to set. See @ref WT_TX_TXN_TYPE for available
+	 * options.
+	 * Durable timestamp should be set only after setting the commit timestamp.
+	 * @param ts the timestamp. Zero means none.
```

Comment:
Maybe `zero clears any previously set timestamp`?

Reply:
Thanks, that's a bit clearer. I'll wait for the "does zero ever make sense" thread to finish up before changing this.

Reply:
Fixed in 282b710